File: 프로그래머스/2/12980. 점프와 순간 이동/점프와 순간 이동.java
class Solution {
    public int solution(int n) {
        int answer = 0;

        while (n != 0) {
            int a = n / 2;
            int b = n % 2;
            n = a;

            if (b == 1) {
                answer++;
            }
        }

        return answer;
    }
}

File: 프로그래머스/2/12980. 점프와 순간 이동/점프와 순간 이동.js
function solution(n) {
    let answer = 0;

    while (n !== 0) {
        const a = Math.floor(n / 2);
        const b = n % 2;
        n = a;

        if (b === 1) {
            answer++;
        }
    }

    return answer;
}